dev/releases/HelpLinks-to-JSON.g

```diff
@@ -0,0 +1,61 @@
+##  A utility for dumping links to all available manual sections
+##  (for use with named manual references on web pages or in search utilities).
+##
+##  The main purpose for this script is to create the help-links.json file that
+##  is used by the GAP website. This happens as part of the release process.
+##
+##  In order to be used on the GAP website, relative to a particular GAP
+##  version, this script should be run from within the corresponding released
+##  version of GAP, with exactly its bundled packages (installed in the pkg/
+##  subdirectory), and with no special user config. The GAP and package manuals
+##  are precompiled in the released version.
+
+LoadPackage("json");
+
+out := OutputTextUser();
+r := rec();
+# load all books (without prompting the user for input)
+Perform(HELP_KNOWN_BOOKS[1], HELP_BOOK_INFO);;
+for x in NamesOfComponents(HELP_BOOKS_INFO) do
+  book := HELP_BOOKS_INFO.(x);
+
+  # <x> is the lowercase bookname; we want two capitalised versions:
+  # - One with " (not loaded)" removed (we don't want this string in the end)
+  # - One with ": " added (for matching and removing this as a substring)
+  bname := ReplacedString(book.bookname, " (not loaded)", "");
+  match := Concatenation(book.bookname, ": ");
+
+  r.(bname) := rec();
```

Comment:
Another development to make diffs smaller, and to reduce file size, could be to separate the directory name of the package from the path. Therefore, instead of `r.Digraphs` looking like:
```
"Digraphs": {
  "String": "pkg/digraphs-1.4.1/doc/chap9.html#X81FB5BE27903EC32",
  "CharacteristicPolynomial": "pkg/digraphs-1.4.1/doc/chap5.html#X87FA0A727CDB060B",
  "AsTransformation": "pkg/digraphs-1.4.1/doc/chap3.html#X7C5360B2799943F3",
  "Copyright": "pkg/digraphs-1.4.1/doc/chap0.html#X81488B807F2A1CF1",
  ...
}
```
(with lots of repeated `pkg/digraphs-1.4.1`; meaning also that every single line of every single package will change when a new version of the package is picked up), we could instead have:
```
"Digraphs": {
  "directory": "pkg/digraphs-1.4.1/"
  "reference": {
    "String": "pkg/digraphs-1.4.1/doc/chap9.html#X81FB5BE27903EC32",
    "CharacteristicPolynomial": "pkg/digraphs-1.4.1/doc/chap5.html#X87FA0A727CDB060B",
    "AsTransformation": "pkg/digraphs-1.4.1/doc/chap3.html#X7C5360B2799943F3",
    "Copyright": "pkg/digraphs-1.4.1/doc/chap0.html#X81488B807F2A1CF1",
    ...
  }
}
```
I think this would require only a small change to the website to accommodate.

Reply:
I've done this.